Sources/Sentry/Public/SentryTransaction.h

```diff
@@ -58,7 +58,7 @@ SENTRY_NO_INIT
  */
 
 - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transactionContext
-                                    andHub:(SentryHub *_Nullable)hub;
+                                       hub:(nullable SentryHub *)hub;
```

Comment:
`hub` should be non nullable, otherwise, transaction should not even be created

Reply:
We dont have NoOpHub in obj-c yet.

Reply:
For v1 can we just always instantiate one with sampled=false?
It's what we did in .NET to move faster

Reply:
that's ok and wfm, although the comment is about the `hub` nullability